README 2.md

```diff
@@ -0,0 +1 @@
+# Math-Magicians-
```

Comment:
- [ ] [OPTTIONAL] kindly remove this readme file as you already have another readme file which is up to date

Reply:
I didn't notice that one I have no idea when I generated that, I'll delete it thanks!